app/presenters/finder_presenter.rb

```diff
@@ -93,7 +93,7 @@ def email_alert_signup_url
   def facet_details_lookup
     @facet_details_lookup ||= begin
       result_hashes = @facet_hashes.map do |facet|
-        facet_name = facet['name']
+        facet_name = facet.fetch('short_name', facet['name'])
```

Comment:
I think I've missed something, why can you use `short_name` and not have to use `facet['short_name']`?

Reply:
`Hash#fetch` allows a second arg as a fallback if the key specified isn't found.
In short this is like doing

```ruby
facet['name'] unless facet.key?('short_name')
```